doc/adrs/053_add_support_for_form_descriptions_in_the_view_converter.adoc

```diff
@@ -0,0 +1,43 @@
+= ADR-053 - Add support for form descriptions in the view converter
+
+== Context
+
+The RepresentationDescriptions in the View DSL needs to be converted in the supported type `org.eclipse.sirius.components.representations.IRepresentationDescription`.
+For now, only `DiagramDescription` are converted by the `ViewConverter`.
+
+== Decision
+
+We need to create and add a `FormDescriptionConverter` to the `ViewConverter`.
+The `ViewConverter.convert()` method statically converts DiagramDescription:
+
+```            
+result = view.getDescriptions().stream()
+    .filter(org.eclipse.sirius.components.view.DiagramDescription.class::isInstance)
+    .map(org.eclipse.sirius.components.view.DiagramDescription.class::cast)
+    .map(viewDiagramDescription -> this.convertDiagramDescription(viewDiagramDescription, interpreter))
+    .collect(Collectors.toList());
+                         
+```
+
+We need to modify this logic to dispatch to the correct converter according to the `IRepresentationDescription` concrete type.
+We will create a common interface `IRepresentationDescriptionConverter` for `FormDescriptionConverter` and `DiagramDescriptionConverter`. The `ViewConverter` will no longer reference a specific converter implementation.
```

Comment:
Use a new line for each sentence (it helps when reading diffs).

Reply:
ok